src/benchmarks/micro/MicroBenchmarks.sln

```diff
@@ -6,12 +6,11 @@ MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MicroBenchmarks", "MicroBenchmarks.csproj", "{D99F63AE-3154-4F13-9424-FA5F9D032D1D}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BenchmarkDotNet.Extensions.Tests", "..\..\tests\harness\BenchmarkDotNet.Extensions.Tests\BenchmarkDotNet.Extensions.Tests.csproj", "{0A02797B-B016-4F5C-AADB-11EE2653BA92}"
-	ProjectSection(ProjectDependencies) = postProject
```

Comment:
I have added this VS solution build order workaround a long time ago, now it's not needed